src/components/Footer/Footer.js

```diff
@@ -54,6 +49,15 @@ const Footer = () => {
           format={(f) => `${t('Geodaten')} ${f.join(', ')}`}
           className="tm-copyright"
         />
+        <ScaleLine map={map} />
+        <ProjectionSelect projections={projections} />
```

Comment:
can you add a bit more width for the projection selction  swiss projection are not displayed  in one line

Reply:
Sure :)